src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -260,6 +260,11 @@ protected override async Task StartAsyncCore(CancellationToken cancellationToken
         public async Task InitializeAsync()
         {
             _stopwatch.Start();
+            if (!_environment.IsPlaceholderModeEnabled())
+            {
+                string runtimeLanguage = string.IsNullOrEmpty(_currentRuntimelanguage) ? "none" : _currentRuntimelanguage;
```

Comment:
This isn't telling us the number of functions in that language, right? Just whatever the app setting was?

@ColbyTresness -- Can you extract the details you want just by having this? I know @fabiocav wanted to make sure we could track mixed-language apps, which isn't technically supported, but may still happen.

Reply:
Right, this is just telling us what language the application was configured to run.

Reply:
This isn't a count of the number of functions for a particular runtime - it's a count of the number of times a host has started/restarted for a particular language. This metric wouldn't make sense for multi-language hosts. If we wanted to handle that, we'd instead have a function.runtime.{runtime} metric, similar to the existing function.binding.{bindingType} metric which provides counts per binding on startup.

Reply:
I think that, particularly after some recent work with the out-of-proc workers, mixed language environments will no longer be something we need to consider, so this work.

Reply:
Right -- I want to make sure that this is what @ColbyTresness and @jeffhollan need. I thought someone mentioned wanting the count of functions-per-language (or maybe it was total-executions-per-language?). I'm getting all the conversations mixed up in my head :-) 

As long as they're good with the metric and have a way to pull what they need, this looks good to me.

Reply:
You're remembering the conversation we had. With the more strict behavior where mixed runtimes are not supported, having this as is will also allow the function counts to be derived, so I think this is sufficient.